layer/compile_time_layer.cc

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 #include <cassert>
+#include <chrono>
```

Comment:
nit: no need to include chrono here, because the file does not directly name anything in chrono. The type behind `DurationClock` will introduce the necessary includes.